config/example.json

```diff
@@ -44,7 +45,13 @@
     "name": "aws",
     "aws": {
       "region": "us-east-1",
-      "lambda-role": ""
+      "lambda-role": "",
+      "docker_registry": {
+      "registry": "",
```

Comment:
What is the difference between `registry` and `docker_registry`?

Reply:
docker_registry is just the key and it has three values : 
registry: the registry name ( for aws it will be something like : f"{account_id}.dkr.ecr.{region}.amazonaws.com" 
username: ECR username 
password: ECR password

Reply:
You're right, my bad! :-)